README.md

```diff
@@ -123,8 +123,9 @@ A note on abbreviations: `CC` is ComputerCraft, `CC:T` is ComputerCraft: Tweaked
 - [Recrafted](https://github.com/ocawesome101/recrafted) -  A rewrite of CraftOS aiming for full feature parity while maintaining "saner API design."
 - [UnBIOS](https://gist.github.com/MCJack123/42bc69d3757226c966da752df80437dc) -  A program to "undo" CraftOS, returning the system to the same state it was in before running the BIOS. Useful for OS development.
 
-### Economy
+### Economy Lua programs
```

Comment:
Why was this changed?

Reply:
Like i said earlier, we currently have two `### Economy` categories, making it very confusing. We can use a different name _(or change the name of the other category, or even fuse them?)_. But just having it twice is not a good solution I thought, which is why I changed it.

Reply:
After taking a closer look, I think we should remove the 'Economy' section in 'non-Lua programs' and place 'Krist Server' there, with the two other Krist-related items placed as sub-items of the Krist Server item.

Reply:
Added in latest commit.